website/docs/cloud-docs/api-docs/policy-evaluations.mdx

```diff
@@ -0,0 +1,296 @@
+---
```

Comment:
The header for this page is showing lowercase

![image](https://github.com/hashicorp/terraform-docs-common/assets/5819622/9b2f2391-64e5-43be-9567-892b82aacb3d)